modules/virt-switching-displays.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
+
+:_content-type: PROCEDURE
+[id="virt-switching-displays_{context}"]
+= Switching between VM displays
+
+If your virtual machine (VM) has multiple displays, you can view each one by switching displays in the web console.
+
+.Prerequisites
+
+* The mediated device is configured in the `HyperConverged` custom resource and assigned to a VM.
+
+.Procedure
+
+. In the OpenShift Virtualization console, click *Workloads* -> *Virtualization* from the side menu.
+. Click the *Virtual Machines* tab.
+. Select a Windows virtual machine to open the *Virtual Machine Overview* screen.
```

Comment:
the title of the page is "Overview"
```suggestion
. Select a Windows virtual machine to open the *Overview* screen.
```
![Overview](https://user-images.githubusercontent.com/2181522/177036140-eaec371c-25be-496d-b024-2014a4595992.png)